.golangci.yaml

```diff
@@ -57,6 +57,9 @@ issues:
       - staticcheck
       # TODO(#768): Drop when incrementing the API version.
       text: "SA1019: constants.QueueAnnotation is deprecated"
+    - linters:
+        - fatcontext
+      path: "^test/*"
```

Comment:
We can skip this linter for integration and e2e tests because we are using nested contexts like here:

https://github.com/kubernetes-sigs/kueue/blob/b508202ee890903cf985cb5da1afe7cdebda837f/test/integration/scheduler/suite_test.go#L63

Reply:
so, is this change required, or we can drop?

Reply:
Yes, we should add it to fix https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/3061/pull-kueue-verify-main/1835622466400030720.

Reply:
This changes to skip this errors.

Reply:
I see, thanks